src/js/extend.js

```diff
@@ -40,19 +40,17 @@ const _inherits = function(subClass, superClass) {
  * Function for subclassing using the same inheritance that
  * videojs uses internally
  *
- * @static
- * @const
+ * @function
+ * @param    {Object} superClass
+ *           The class to inherit from
  *
- * @param {Object} superClass
- *        The class to inherit from
- *
- * @param {Object} [subClassMethods={}]
- *        The class to inherit to
+ * @param    {Object} [subClassMethods={}]
+ *           The class to inherit to
  *
- * @return {Object}
- *         The new object with subClassMethods that inherited superClass.
+ * @return   {Object}
+ *           The new object with subClassMethods that inherited superClass.
  */
-const extendFn = function(superClass, subClassMethods = {}) {
+const extend = function(superClass, subClassMethods = {}) {
```

Comment:
This mis-match in name between what was exposed on `videojs` and what was here in the module looked as if it was confusing. This should be a non-functional change.

Reply:
👍

Reply:
We should probably think about just getting rid of this in some future version too.

Reply:
Don't we still need it for allowing people to extend built-in classes? Maybe once `class` is supported universally we won't need it.

Reply:
Oh, sorry, I meant that `extend` was `extendFn` probably because of IE8 issues. Yeah, makes sense that we still need it.